tests/smoke/package.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
I wonder if it would have been doable to not have to copy paste the same project three times. Maybe by having a single test project, where between each jest call you just npm install another react version?